src/VisualStudio/CSharp/Test/PersistentStorage/EsentPersistentStorageTests1.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
There was a lot of code moved out of this file.  By renaming the file, the PR shows up effectively in github.  If i don't, it is a mess.